.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,142 @@
+name: "Docker build"
+
+on:
+  workflow_call:
+    inputs:
+      docker_file:
+        type: string
+        required: true
+      image_name:
+        type: string
+        required: true
+      test_services_to_start:
+        type: string
+      test_script:
+        type: string
+      push_image:
+        type: boolean
+      build_platforms:
+        type: string
+        required: true
+      test_success:
+        description: required when pushing images
+        type: boolean
+
+    secrets:
+      dockerhub_username:
+        description: 'Required if the docker image is pushed'
+        required: false
+      dockerhub_token:
+        required: false
+
+jobs:
+  build:
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v3
+      - name: Cache Docker layers
+        uses: actions/cache@v3
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-v1-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-v1-
+      - name: Setup qemu
+        if: contains(inputs.build_platforms, 'arm64')
+        uses: docker/setup-qemu-action@v1.2.0
+        with:
+          platforms: arm64
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1.6.0
+        with:
+          driver-opts: |
+            network=host
+      - uses: docker/login-action@v1.14.1
+        if: ${{inputs.push_image}}
+        with:
+          username: ${{ secrets.dockerhub_username }}
+          password: ${{ secrets.dockerhub_token }}
+
+      - name: Docker meta
+        id: docker_meta_success
+        uses: docker/metadata-action@v3.6.2
+        if: ${{inputs.push_image}}
+        with:
+          images: ${{inputs.image_name}}
+          tags: |
+            type=sha
+            type=ref,event=branch
+            type=semver,pattern=v{{version}}
+            type=raw,value=dev,enable=${{ inputs.test_success }}
+            type=raw,value=debug,enable=${{ !inputs.test_success }}
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) && startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
```suggestion
            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) && startsWith(github.ref, 'refs/tags/v') }}
            type=semver,pattern={{major}}.{{minor}},enable=${{ !startsWith(github.ref, 'refs/tags/v0.0.') }}
            type=semver,pattern={{major}},enable=${{ !startsWith(github.ref, 'refs/tags/v0.') }}
```

Reply:
I'm adding two last lines, not changing the first one.

Pushes Docker Hub tags `1` or `1.1`, but not major `0` or `0.1`